app/helpers/application_helper/button/vm_webmks_console.rb

```diff
@@ -14,7 +14,8 @@ def disabled?
   end
 
   def supported_vendor_api?
-    @record.host.vmm_version.to_f >= min_supported_api_version
+    return @record.host.vmm_version.to_f >= min_supported_api_version unless @record.host.nil?
```

Comment:
Does it make sense to enable the console for VMs that have no host? I'd say no.

Reply:
@martinpovolny :+1:

Reply:
I have to agree.  I also updated the other ```vmm_version``` method call as well.